console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java

```diff
@@ -37,28 +38,30 @@
  */
 @ControllerAdvice
 public class ConsoleExceptionHandler {
-    
+
     private static final Logger LOGGER = LoggerFactory.getLogger(ConsoleExceptionHandler.class);
-    
+
     @ExceptionHandler(AccessException.class)
     private ResponseEntity<String> handleAccessException(AccessException e) {
         LOGGER.error("got exception. {}", e.getErrMsg());
         return ResponseEntity.status(HttpStatus.FORBIDDEN).body(e.getErrMsg());
     }
-    
+
     @ExceptionHandler(IllegalArgumentException.class)
     private ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException e) {
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(ExceptionUtil.getAllExceptionMsg(e));
     }
-    
+
```

Comment:
Please don't change the indent. Use nacos code style to reformat codes.

Reply:
Handled